cargo-verify/src/main.rs

```diff
@@ -413,13 +421,13 @@ fn verify(opt: &Opt, package: &str, target: &str) -> CVResult<Status> {
     let end = Instant::now();
 
     println!(
-        "test result: {:#}. {} passed; {} failed",
+        "\ntest result: {:#}. {} passed; {} failed",
```

Comment:
The `\n` at the beginning needs to be documented. I'm guessing it's here to close the `print!` from `verifier_run` in the -q case?

Reply:
Good idea - done